Vagrantfile

```diff
@@ -350,6 +350,9 @@ Vagrant.configure('2') do |config|
     # Reference: https://github.com/chef/vagrant-omnibus/issues/90#issuecomment-51816397
     cfg.vm.guest = :windows
     default_omnibus config
+    # config below prevents the installation of latest Chef on the box.
+    # Reference: https://github.com/chef/vagrant-omnibus/issues/129
+    config.omnibus.install_url = 'https://packages.chef.io/files/stable/chef/12.17.44/windows/2012r2/chef-client-12.17.44-1-x64.msi'
```

Comment:
Can we use 12.18.31?  That should be the latest working version wrt windows.

Reply:
c56339bcc3223c7413cf699102abb7237d3f8049